src/PolygonEcosystemToken.sol

```diff
@@ -26,9 +29,11 @@ contract PolygonEcosystemToken is ERC20Permit, AccessControlEnumerable, IPolygon
         _grantRole(DEFAULT_ADMIN_ROLE, governance);
         _grantRole(EMISSION_ROLE, emissionManager);
         _grantRole(CAP_MANAGER_ROLE, governance);
+        _grantRole(PERMIT2_REVOKER_ROLE, governance);
```

Comment:
This needs to be different than cap manager bcz cap manager is time locked, this isnt.